clients/cpp-console/src/common/DialogManager.cpp

```diff
@@ -262,14 +261,20 @@ void DialogManager::ResumeKws()
 
 void DialogManager::StartListening()
 {
-    log_t("Now listening...");
     if (_bargeInSupported)
     {
         _player->Stop();
     }

```

Comment:
In all cases stop playback per my comment above. Sorry for not catching that before

Reply:
Updated to remove "if (_bargeInSupported)" block.